scss/field_formatter.scss

```diff
@@ -0,0 +1,25 @@
+.react-paragraphs-row {
+  [data-react-columns] {
+    width: 100%;
+  }
+}
+
+@media (min-width: 768px) {
+  .react-paragraphs-row {
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: space-between;
```

Comment:
What was the plan to have the gutters match the different gutter sizes at our theme's breakpoints? 

https://github.com/SU-SWS/decanter/blob/master/core/src/scss/utilities/variables/core/_gutter.scss

Reply:
Either override the template or add a reprocess to add appropriate classes where they are needed

Reply:
or just target the classes and add our own styles in `stanford_profile_styles`

Reply:
Add a ticket in our backlog so we don't forget.

Reply:
https://stanfordits.atlassian.net/browse/D8CORE-1679
Bare bones ticket for now but we can add to it if its not obvious from this conversation.